morpheus/_lib/include/morpheus/stages/filter_detection.hpp

```diff
@@ -84,13 +84,18 @@ class FilterDetectionsStage : public mrc::pymrc::PythonNode<std::shared_ptr<Mult
      * @param threshold : Threshold to classify
      * @param copy : Whether or not to perform a copy default=true
      */
-    FilterDetectionsStage(float threshold, bool copy = true);
+    FilterDetectionsStage(float threshold,
+                          bool copy               = true,
+                          FilterSource operate_on = FilterSource::AUTO,
```

Comment:
Not a huge fan of `operate_on`. We dont have any other arguments that use an active verb like this. Can we just keep it `source`, data source, or column source or similar?